Source/DafnyCompilers/CompilerCommon.dfy

```diff
@@ -962,272 +1005,6 @@ module {:extern "DafnyInDafny.Common"} DafnyCompilerCommon {
     // Both implementations of Deep should work, but NonRec can be somewhat
     // simpler to work with.  If needed, use ``DeepImpl.Equiv.All_Expr`` to
     // switch between implementations.
-    module Deep refines DeepImpl.Rec {}
-  }
-
-  module Rewriter {
```

Comment:
Can we avoid splitting files in this PR?  I'd like to preserve the history, and this requires an explicit merge commit when the move is done.